fix: change wording for spec file creation card

Manuel-Suarez-Abascal opened the pull request:
<!-- Thanks for contributing! PLEASE...
- Read our contributing guidelines: https://github.com/cypress-io/cypress/blob/develop/CONTRIBUTING.md 
- Read our Code Review Checklist on coding standards and what needs to be done before a PR can be merged: https://github.com/cypress-io/cypress/blob/develop/CONTRIBUTING.md#Code-Review-Checklist
- Mark this PR as "Draft" if it is not ready for review.
- Make sure you set the correct base branch based on what packages you're changing: https://github.com/cypress-io/cypress/blob/develop/CONTRIBUTING.md#branches
-->

- Closes [issue-21908](https://github.com/cypress-io/cypress/issues/21908)

### User-facing changelog

Fix misleading wording when creating a new starter spec file in the UI.

### Additional details

N/A

### Steps to test

- Launch Cypress's test runner.
- Click on the button `New Spec` button.
- You should see the new wording in the cards to create a new starter spec file.

### How has the user experience changed?

**Before changes:**

<img width="813" alt="Screenshot 2022-11-26 at 6 06 16 AM" src="https://user-images.githubusercontent.com/32891808/204085736-d7b53a19-dd0a-4957-a36e-688850d8e283.png">

**After changes:**

<img width="736" alt="Screenshot 2022-11-26 at 6 11 22 AM" src="https://user-images.githubusercontent.com/32891808/204085765-7f565105-c916-4ca2-858c-83bd9730d296.png">

### PR Tasks


- [X] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? <!-- Link to PR here -->
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

Commit:
change wording for spec file creation card

Commit:
fix typo

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/cypress-io/cypress?pullRequest=24825) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/cypress-io/cypress?pullRequest=24825) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: Manuel-Suarez-Abascal<br/>:x: Manuel Abascal<br/><hr/>**Manuel Abascal** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/cypress-io/cypress?pullRequest=24825) it.</sub>

Commit:
fix: fix misleading wording

Manuel-Suarez-Abascal committed:
Merge branch 'develop' into issue-14528

Commit:
fix: revert some unwanted changes

Manuel-Suarez-Abascal committed:
Fix: revert unwanted changes

Manuel-Suarez-Abascal committed:
Fix: removed empty spacing

Manuel-Suarez-Abascal committed:
Fix: remove line

Manuel-Suarez-Abascal committed:
fix: remove empty space

marktnoonan commented:
I like the phrase "starter spec". I think we could make the title "Create starter spec"  to avoid wrapping and make it simpler.

Commit:
fix: address PR comment

Commit:
fix: solve merge conflicts

Manuel-Suarez-Abascal commented:
> I like the phrase "starter spec". I think we could make the title "Create starter spec" to avoid wrapping and make it simpler.

@marktnoonan sounds good, I have addressed the comments.

Manuel-Suarez-Abascal committed:
Merge branch 'develop' into issue-14528

Manuel-Suarez-Abascal committed:
Merge branch 'develop' into issue-14528

lmiller1990 commented:
Looks good to me. Our CLA is complaining: https://github.com/cypress-io/cypress/pull/24825#issuecomment-1328026970

Can you take a look and see if you can get the commits associated with a GH user that has signed the CLA @Manuel-Suarez-Abascal? Once that's done, we can review and merge.

emilyrohrbough commented on packages/frontend-shared/src/locales/en-US.json:
```diff
@@ -94,8 +94,8 @@
       "importEmptySpec": {
-        "header": "Create new empty spec",
-        "description": "We'll generate an empty spec file which can be used to start testing your application.",
+        "header": "Create starter spec",
```
```suggestion
        "header": "Create new spec",
```

Manuel-Suarez-Abascal replied:
Hello @emilyrohrbough, I have submitted another pull request to address the Contributor License Agreement (CLA) issue.

In response to the comments on the PR, I have used the term 'starter' instead of 'new spec' as suggested in this issue comment (https://github.com/cypress-io/cypress/issues/21908#issuecomment-1154792807). 

Can you please share your thoughts on this? Thank you.

emilyrohrbough commented on packages/frontend-shared/src/locales/en-US.json:
```diff
@@ -94,8 +94,8 @@
-        "header": "Create new empty spec",
-        "description": "We'll generate an empty spec file which can be used to start testing your application.",
+        "header": "Create starter spec",
+        "description": "We'll generate a starter spec file which can be used to start testing your application.",
```
```suggestion
        "description": "We'll generate a template spec file which can be used to start testing your application.",
```

Manuel-Suarez-Abascal replied:
Same comment as [here](https://github.com/cypress-io/cypress/pull/24825/files/cb11c3c3d520451f71dc5aadb59068dd41105f3a#r1056628681).

emilyrohrbough commented on packages/frontend-shared/src/locales/en-US.json:
```diff
@@ -110,7 +110,7 @@
       },
       "importEmptySpec": {
         "header": "Create a new spec",
```
```suggestion
        "header": "Create new spec",
```

emilyrohrbough commented on packages/frontend-shared/src/locales/en-US.json:
```diff
@@ -110,7 +110,7 @@
       "importEmptySpec": {
         "header": "Create a new spec",
-        "description": "We'll generate an empty spec file to start testing components.",
+        "description": "We'll generate a starter spec file to start testing components.",
```
```suggestion
        "description": "We'll generate a template spec file which can be used to start testing your components.",
```
The verbiage between component & e2e testing ideally will be pretty similar.

Manuel-Suarez-Abascal replied:
Same comment as [here](https://github.com/cypress-io/cypress/pull/24825/files/cb11c3c3d520451f71dc5aadb59068dd41105f3a#r1056628681).

Manuel-Suarez-Abascal commented:
**PR closed in favor of:** https://github.com/cypress-io/cypress/pull/25271 in order to fix the CLA issue. cc: @lmiller1990
